src/components/ContactList.jsx

```diff
@@ -0,0 +1,21 @@
+import ContactListItem from "./ContactListItem.jsx";
+function ContactList({ contacts }) {
+  return (
+    <>
+      <section>
+        <h2>Contacts</h2>
+        <ul>
+          {contacts && contacts.length > 0 ? (
```

Comment:
`contacts.length > 0` this is enough

Reply:
I guess you are not using propTypes so you need to check if contacts has been assigned